data/databases.md

```diff
@@ -26,6 +26,10 @@
 - [Time-scale](https://www.timescale.com/)
 - [kdb+](https://en.wikipedia.org/wiki/Kdb%2B) - is a column-based relational time series database (TSDB) with in-memory (IMDB) abilities, developed and marketed by [Kx Systems](https://kx.com/)
 
+## Misc.
```

Comment:
You could use the title SQL as these are SQL specific links but you can do it in the next PR

Reply:
SQL as a subheading can work too.